.gitignore

```diff
@@ -6,6 +6,15 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
+# yarn v2 files
```

Comment:
I have updated Yarn to the latest version and tested it thoroughly to make sure it is working correctly. The new Yarn version requires these files to be ignored in Git.

Reply:
This is amazing! Thanks so much for upgrading to Yarn4 🥇 So much faster!!